### ISSUE TITLE
Error code T028 error only applies to Jinja and should be J028

### ISSUE THREAD

**Issue by krukas** [Riverside-Healthcare/djLint#178]
T028 is not a general template error and should have the prefix for Jinja

**Comment by christopherpickering** [comment]
Good point, thats valid in `njk` as well, but I don't think twig. Instead of duplicating the rule for `njk` I will update the exclude list for the other profiles.

Thanks!

**Comment by christopherpickering** [comment]
Oh, looks like Django was already excluded. What template lang are you using?
![image](https://user-images.githubusercontent.com/17788706/152174529-e89c8910-987c-4b3f-b04a-e14332ed558b.png)

**Comment by krukas** [comment]
Add the beginning I had no profile (I thought the default is Django, couldn't find what default is) set and was getting this error. I changed the profile to Django and it error disappears.

I still created this issue because it was not a generic template error, and with some google i stumble upon Jinja and that's why I though an Jinja only feature.

I don't know what is cleaner, but maybe you can extend rules. In that way you can apply this rule for Jinja and njk with own prefix?

**Comment by christopherpickering** [comment]
Yeah, that's probably a better idea.. just enable the generic rules by default, then other specific rules when profiles are set.
